scripts/docker/postgres/init.sh

```diff
@@ -13,4 +13,4 @@ make && make install
 # Install neo4j fdw
 echo "~~~~~~~~~~~~~ Installing Neo4j FDW"
 cd /neo4j-fdw/source/
-make install
+make install
```

Comment:
In bash script the good practice is to have an empty new line at the end of the file

Reply:
This must have accidentally made it into my merge.